src/confdb/confdb.c

```diff
@@ -1087,7 +1087,12 @@ static errno_t confdb_init_domain(struct sss_domain_info *domain,
 
     tmp = ldb_msg_find_attr_as_string(res->msgs[0], CONFDB_DOMAIN_AUTO_UPG, NULL);
     if (tmp == NULL || *tmp == '\0') {
-        tmp = "false";
+        if (domain->provider != NULL
+                && strcasecmp(domain->provider, "idp") == 0) {
```

Comment:
Currently man 'sssd.conf' man page doesn't talk about default value for non-subdomains at all.
Is it worth mention?

And maybe `auto_private_groups = true` should be removed from examples in 'sssd-idp' man page?